about/our-web-team.php

```diff
@@ -120,18 +161,53 @@
+                                                    height="404" class="img-responsive"/>
                         </div>
                         <div class="cell-md-6">
                             <div class="inset-xl-right-70 inset-xl-left-70 inset-left-15 inset-right-15">
-                                <h6 id="aryan-damle" class="h6-with-small"><a href="#aryan-damle"> Aryan Damle</a><span class="small text-silver-chalice">Team Member</span></h6>
-                                <p>Aryan Damle is a Computer Science student at UNT. He is an aspiring full stack web developer and an avid Home Assistant enthusiast. He mentors a high school robotics team and loves to work on robots in his free time. You can find him at your local car meet on weekends if he isn't busy working on a robot or fixing something in his garage.</p>
+                                <h6 id="aryan-damle" class="h6-with-small"><a href="#aryan-damle"> Aryan Damle</a><span
+                                            class="small text-silver-chalice">Team Member</span></h6>
+                                <p>Aryan Damle is a Computer Science student at UNT. He is an aspiring full stack web
+                                    developer and an avid Home Assistant enthusiast. He mentors a high school robotics
+                                    team and loves to work on robots in his free time. You can find him at your local
+                                    car meet on weekends if he isn't busy working on a robot or fixing something in his
+                                    garage.</p>
                                 <ul class="list-inline-lg">
                                     <li>
-                                        <a href="https://www.linkedin.com/in/aryan-damle-8691b11bb/" class="icon icon-sm text-primary fa-linkedin"></a>
+                                        <a href="https://www.linkedin.com/in/aryan-damle-8691b11bb/"
+                                           class="icon icon-sm text-primary fa-linkedin"></a>
                                     </li>
                                     <li>
-                                        <a href="https://github.com/aryan-damle" class="icon icon-sm text-primary fa-github"></a>
+                                        <a href="https://github.com/aryan-damle"
+                                           class="icon icon-sm text-primary fa-github"></a>
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <!-- Mary Plana -->
+                <div class="cell-lg-12 bio-area">
+                    <div class="range range-sm-middle">
+                        <div class="cell-md-3"><img src="/images/bio-pics/mary-plana.jpg" alt="" width="360" height="404"
```

Comment:
should store image in web-team-pics folder

Reply:
He means `/images/web-team-pics/mary-plana.jpg`